package.json

```diff
@@ -8,12 +8,15 @@
     "eslint-plugin-node": "^5.1.0",
     "eslint-plugin-promise": "^3.5.0",
     "eslint-plugin-standard": "^3.0.1",
-    "grunt-contrib-connect": "~0.9",
     "grunt-documentation": "*",
-    "grunt-mocha": "^1.0.2",
+    "grunt-contrib-connect": "~0.9",
+    "grunt-execute": "^0.2.2",
     "grunt-plato": "~0.2.1",
+    "bluebird": "*",
```

Comment:
Needed to add bluebird as a dev dependency for node execution, otherwise it cannot resolve bluebird, weird as it is being added as external, but does not seem to work.